requests_toolbelt/utils/formdata.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""Implementation of nested form-data encoding function(s)."""
+from .._compat import basestring
+from .._compat import urlencode as _urlencode
+
+
+__all__ = ('urlencode',)
+
+
+def urlencode(query, doseq=False, safe='', encoding=None, errors=None):
```

Comment:
Given that you don't touch the other arguments directly, is there any reason this shouldn't be `def urlencode(query, *args, **kwargs)`? This will allow you to avoid the `try..except` block further down the page.